runscanner-illumina/README.md

```diff
@@ -0,0 +1,17 @@
+# Illumina Support for Run Scanner
+To use the Illumina support in Run Scanner, this additional package must be built.
+
+You will need a copy of [jsoncpp](https://github.com/open-source-parsers/jsoncpp) and an AutoTools-capable build environment. On Debian/Ubuntu:
```

Comment:
Yes. I spent several days fighting with that plugin, but it's really not capable of handling the Illumina library as it's built.